README.md

```diff
@@ -8,39 +10,25 @@
 
 *The graph database for delightful React state management*
 
-## Installation
-
-```bash
-# NPM
-npm install homebase-react --save
-
-# Yarn
-yarn add homebase-react
-```
-
-## Purpose
-
-Data should be a first class citizen on the client. We deserve the ergonomics and power of relational databases not just on the server, but in our React applications.
-
-Homebase React lets you plug a relational database into your React application with just 3 lines of code. In fact, it's the same database that powers Roam Research and many other ClojureScript applications, but with an API that's familiar to React and JS developers.
 
-- Replace Redux with something simpler and more declarative
-- Stop spending time wiring up custom datatypes, reducers, caches and other bespoke state mumbo jumbo
-- Gain all the benefits of relational data like a single source of truth
-- Traverse your data graph like it's one big JSON object
-- Query your database with a convenient JSON query syntax
-- Query your database with Clojure style [Datalog](https://docs.datomic.com/on-prem/query.html) if you need more power
-- It's just data
+Homebase React makes state management painless by enabling you to plug a relational graph database into your React application with just 3 lines of code. This is the same database that powers Roam Research and many other ClojureScript applications, but with an API that's familiar to React and JS developers.
```

Comment:
A bit more Hemingway'd

Homebase React makes state management painless. Plug a graph database into your React application with 3 lines of code. This is the same database that powers Roam Research and many other ClojureScript applications, but with an API that works for React and JS developers.

Reply:
I don't think it's obvious that the first two sentences are connected.

Reply:
I don't think it's obvious to people that using a database makes state management easier. So I try to posit that idea here and then prove it with the features section.

Reply:
I can also try to make the sentence with connected ideas shorter if that's what you don't like. I'm personally not a Hemmingway fan, but I am a Stephen King fan.